### PR TITLE
Update instructions for installation of pre-requisites for new role based authentication

### PR DESCRIPTION
**Pull request by neilbmclaughlin**

### CODE REVIEW

File: docs/developer-laptop-setup/install-kubectl.md [hunk 1]
```diff
@@ -39,6 +45,10 @@ mkdir ~/.kube \
 ## MacOS Specific config
 TBC
 
+## EKS Configuration
+
+Cluster specfic EKS `kubctl` configuration files can be downloaded [from](http://gitlab.ffc.aws-int.defra.cloud/kubernetes) this private GitLab repo.
```
Comment (johnwatson484) [inline docs/developer-laptop-setup/install-kubectl.md]:
```suggestion
Cluster specfic EKS `kubectl` configuration files can be downloaded [from](http://gitlab.ffc.aws-int.defra.cloud/kubernetes) this private GitLab repo.
```

**Review by johnwatson484 — DISMISSED** [review]
Just a minor spelling error.  Other than that, I was able to follow the instructions and validate I can receive a token.

File: docs/developer-laptop-setup/install-kubectl.md (hunk 1 above)
Comment (neilbmclaughlin) [reply docs/developer-laptop-setup/install-kubectl.md]: Good spot, fixed.

**Comment by neilbmclaughlin** [comment]
@johnwatson484 - it seems I should requested 're-review' rather than 'dismiss', could you approve please.

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**